cmd/saveWeekly.go

```diff
@@ -86,7 +90,11 @@ func getPlaylistDate() string {
 	return fmt.Sprintf("%d/%d/%d", date.Month(), date.Day(), date.Year())
 }
 func deduplicatePlaylist(playlistName string, user string) bool {
-	searchResults, err := conf.Client.Search(playlistName, spotify.SearchTypePlaylist)
+	if conf.Client == nil {
+		fmt.Println("Client is not initialized")
+		return false
```

Comment:
We might want to do an `os.Exit(1)` here as well, since the case here isn't that the Discover Weekly was not already saved, we weren't able to figure that out if we get to here.